packages/angular/src/utils/mfe/mfe-webpack.ts

```diff
@@ -81,6 +81,21 @@ export function shareWorkspaceLibraries(
   };
 }
 
+function collectPackageSecondaries(pkgName: string, packages: string[]) {
+  const pathToPackage = join(workspaceRoot, 'node_modules', pkgName);
+  const secondaries = readdirSync(pathToPackage)
+    .map((file) => join(pathToPackage, file))
+    .filter((file) => lstatSync(file).isDirectory() && file !== 'node_modules')
```

Comment:
Given we are mapping to `join(pathToPackage, file)`, unless `pathToPackage` is an empty string (which it shouldn't), it doesn't seem `file` here could ever match `node_modules`, isn't it? Should this be using `endsWith` or filtering out `node_modules` before the mapping occurs?

Reply:
yes you're right